composer.json

```diff
@@ -14,15 +14,15 @@
     "ext-mbstring": "*"
   },
   "require": {
-    "php": ">=7.1.0",
+    "php": ">=7.2.0",
     "gridonic/hapi": "dev-master",
-    "symfony/console": "^3.4",
-    "symfony/yaml": "^3.4",
-    "alexkovalevych/jira-api": "^1.0.6",
-    "mnapoli/silly-php-di": "^1.2"
+    "symfony/console": ">=3.4",
+    "symfony/yaml": ">=3.4",
+    "alexkovalevych/jira-api": ">=1.0.6",
+    "mnapoli/silly-php-di": ">=1.2"
```

Comment:
Do we really want to go with *anything* newer than the specified versions? Other libraries usually go with something like `"symfony/console": "^3.4||^4||^5"`.

Reply:
In order to support PHP 7.1 to 7.4, yes...

But I've later dropped 7.1, so maybe not.

Reply:
I've fixed the dependencies to something more conventional.